resources/sdk/clisdkclient/extensions/utils/general.go

```diff
@@ -305,3 +309,19 @@ func MilliSecondsToNanoSeconds(milliSeconds int64) time.Duration {
 func SecondsToNanoSeconds(seconds int) time.Duration {
 	return MilliSecondsToNanoSeconds(int64(seconds)) * 1000
 }
+
+func convertToJSON(data string) string {
+	if strings.EqualFold("json", data_format.InputFormat) && isJSON(data) {
+		return data
+	}
+	result, err := yaml.YAMLToJSON([]byte(data))
```

Comment:
It would be safest to only convert yaml to json if the inputformat flag specifies that